lib/testing/src/lib/process-services-cloud/pages/edit-task-filter-cloud-component.page.ts

```diff
@@ -42,22 +43,26 @@ export class EditTaskFilterCloudComponentPage {
     saveAsButton = element(by.css('[data-automation-id="adf-filter-action-saveAs"]'));
     deleteButton = element(by.css('[data-automation-id="adf-filter-action-delete"]'));
     filter = element(by.css(`adf-cloud-edit-task-filter mat-expansion-panel-header`));
+    candidateGroup = element(by.css('[data-automation-id="adf-cloud-group-search-input"]'));
```

Comment:
is it better use testelment?

Reply:
This element is not used. I later discovered there is already a component for this element: GroupCloudComponentPage
I can remove this one.